### PR TITLE
Improve test coverage

### PR DESCRIPTION
**Pull request by MisterAJ**
This PR:
- Removes `npm` dependency as per [PR#43](https://github.com/rtkwlf/ui/pull/43)

- Adds basic render tests to improve test coverage

### CODE REVIEW

**Comment by rtkaaho** [comment]
Thanks for your contribution :)

**Comment by rtkaaho** [comment]
You will want to resolve the conflicts first :)

**Comment by MisterAJ** [comment]
> You will want to resolve the conflicts first :)

Working on that now

File: src/components/button/__tests__/Button.spec.tsx [hunk 1]
```diff
@@ -60,6 +60,33 @@ describe('Button', () => {
     expect(wrapper.find('StyledButton').prop('customProps').ghost).toBe(true);
   });
 
+  it('renders with loading and ghost props', () => {
+    const wrapper = shallow(
+      <Button loading ghost>Test Button</Button>
+    );
+
+    // @ts-ignore
+    expect(wrapper.find('Icon.Loading'));
```
Comment (rtkaaho) [inline src/components/button/__tests__/Button.spec.tsx]: We will want to be sure that we check if the Icon is there. You can do that by doing something like 
```
 expect(wrapper.exists('Icon.Loading')).toBe(true)
```

That should also remove the need to have the `@ts-ignore`

File: src/components/button/__tests__/Button.spec.tsx [hunk 2]
```diff
@@ -60,6 +60,33 @@ describe('Button', () => {
...
+  });
+
+  it('renders with danger type and loading prop', () => {
+    const wrapper = shallow(
+      <Button type='danger' loading>Test Button</Button>
+    );
+
+    // @ts-ignore
+    expect(wrapper.find('Icon.loading'));
```
Comment (rtkaaho) [inline src/components/button/__tests__/Button.spec.tsx]: same as above

File: src/components/button/__tests__/Button.spec.tsx [hunk 3]
```diff
@@ -60,6 +60,33 @@ describe('Button', () => {
...
+  });
+
+  it('renders with danger type, loading and ghost props', () => {
+    const wrapper = shallow(
+      <Button type='danger' loading ghost>Test Button</Button>
+    );
+
+    // @ts-ignore
+    expect(wrapper.find('Icon.Loading'));
```
Comment (rtkaaho) [inline src/components/button/__tests__/Button.spec.tsx]: same as above

**Pull request merged**